src/lookup/test/lookup.spec.js

```diff
@@ -316,6 +316,29 @@
         });
     });
     
+    describe('Lookup on clear callback', function() {
+        beforeEach(function() {
+            $scope.onClear = jasmine.createSpy('onClear');
+            initDirective('<div dd-lookup ng-model="model" lookup-on-clear="onClear()" url="\'/api/drivers/lookup\'"></div>');
+        });
+        
+       it('is called when clear button is pressed.', function() {            
```

Comment:
Add test to be sure that lookup-on-clear not called on initial load when model is empty